pkg/http/analytics.go

```diff
@@ -36,7 +36,7 @@ func CaptureCommand(command string) ([]byte, Error) {
 		return nil, Error{Err: err, Message: "Unable to generate url"}
 	}
 
-	_, _, resp, err := PutRequest(url, true, map[string]string{"Content-Type": "application/json"}, body)
```

Comment:
😢 I can't believe [I missed this](https://github.com/DopplerHQ/cli/pull/354#discussion_r1085429932)